.env.example

```
@@ -13,4 +13,7 @@ JYOGI_AUTH_URL=
 REDIRECT_URI=
 
 # SSL検証を無効化する場合はfalseに設定（デフォルトはtrue）
-JYOGI_SSL_VERIFY=
\ No newline at end of file
+JYOGI_SSL_VERIFY=
+
+# 管理者のDiscord ID（カンマ区切り）
+ADMIN_DISCORD_IDS=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)